static/js/recent_topics.js

```diff
@@ -36,6 +43,11 @@ exports.process_message = function (msg) {
     return true;
 };
 
+exports.remove_topic = function (stream_id, topic) {
+    const key = stream_id + ':' + topic;
+    topics.delete(key);
+};
```

Comment:
Do we actually want to stop tracking data for muted topics?  I feel like:
(1) If you've muted the topic but then participate in it, you may want the data
(2) If the user unmutes the topic, the data structure updates are a lot more complicated if we delete the data rather than just filtering muted topics when reading from this library.

Reply:
I agree that the updates will be a lot easier if the simply filter the muted topics. 👍